python/mol.cpp

```diff
@@ -176,9 +176,22 @@ void add_mol(py::module& m) {
                      (self.given ? " (" : " (not "), "given)>");
     });
 
+  py::class_<Assembly::Oper>(m, "AssemblyOper")
+    .def(py::init<>())
+    .def_readonly("name", &Assembly::Oper::name)
+    .def_readonly("type", &Assembly::Oper::type)
+    .def_readonly("transform", &Assembly::Oper::transform);
+
+  py::class_<Assembly::Gen>(m, "AssemblyGen")
+    .def(py::init<>())
+    .def_readonly("chains", &Assembly::Gen::chains)
+    .def_readonly("subchains", &Assembly::Gen::subchains)
+    .def_readonly("opers", &Assembly::Gen::opers);
```

Comment:
This would create copy-conversion of `std::vector<Assembly::Oper>`. It might be better to make `std::vector<Assembly::Oper>` bindings opaque. 

Note that declaring `def_readonly` only prohibits assignment to the attribute while it does not stop user from modifying the temporary, e.g. `gen.opers[0].name = ...`.